source/public/index.html

```diff
@@ -45,28 +53,40 @@ <h1>Welcome back to your developer journal!</h1>
         </section>
 
         <section id="projects">
-            <h2>Ongoing Projects</h2>
+            <h2>Projects</h2>
+                 <button class="newProjectButton" title="new project">
+                    + New Project
+                </button>
             <ul id="project-list">
-                <li class="project-item">
+                <!--li--><ul class="project-item">
```

Comment:
Not sure what's going on with the tags here, but `li` should go inside the `ul` tag